lib/env/config.js

```diff
@@ -60,6 +62,13 @@ module.exports = {
       return customConfigContent;
     }
     const configPath = getConfigPath();
-    return Promise.resolve(require(configPath)); // eslint-disable-line
+    try {
+      return Promise.resolve(moduleLoader.require(configPath));
```

Comment:
Hello @nathan-knight ,

I think we are missing an await here to be sure we go in catch.

```suggestion
      return await Promise.resolve(moduleLoader.require(configPath));
```

Reply:
An await here is a no-op because it is returning a promise which will be resolved by the caller

Reply:
It needs the await in the `try {} catch {}` block otherwise a rejection won't get caught.